src/portal/lib/src/service/tag.service.ts

```diff
@@ -140,7 +140,7 @@ export class TagDefaultService extends TagService {
       queryParams = queryParams = new RequestQueryParams();
     }
 
-    queryParams = queryParams.set("detail", "1");
+    queryParams = queryParams.set("detail", "true");
```

Comment:
Seems we don't need this line because by default the api will return details by default?

Reply:
Yes, it's the default behavior, but I think it's clearer to set it implicitly.